src/webargs/aiohttpparser.py

```diff
@@ -147,7 +148,7 @@ def handle_error(
         schema: Schema,
         error_status_code: typing.Union[int, None] = None,
         error_headers: typing.Union[typing.Mapping[str, str], None] = None,
-    ) -> "typing.NoReturn":
```

Comment:
This is actually necessary to account for older patch versions of Python 3.5 https://github.com/marshmallow-code/webargs/issues/366

Reply:
That makes me sad, but makes sense. I'll switch it back.

Reply:
Yeah.. I'm getting kinda exhausted by [these](https://github.com/marshmallow-code/marshmallow/pull/1410) =/

Reply:
Alternatively, we could conditionally install `typing` on Python 3.5. Something like:

```python
'typing>=3.7.4; python_version<"3.5.4"',
```

I think I'm going to end up doing this in marshmallow. 

Doesn't have to happen in this PR though.

Reply:
Disregard that; adding the typing backport doesn't fix it. It's only useful for Python versions less than 3.5.0.